cartography/intel/gcp/iam.py

```diff
@@ -638,116 +638,256 @@ def load_bindings(neo4j_session: neo4j.Session, bindings: List[Dict], project_id
 
         for member in binding['members']:
             if member.startswith('user:'):
+                usr = member[len('user:'):]
+                user = {
+                    'id': f"projects/{project_id}/users/{usr}",
+                    "email": usr,
+                    "name": usr.split("@")[0],
+
+                }
                 attach_role_to_user(
-                    neo4j_session, role_id, f"projects/{project_id}/users/{member[len('user:'):]}",
+                    neo4j_session, role_id, user,
                     project_id, gcp_update_tag,
                 )
 
             elif member.startswith('serviceAccount:'):
+                serviceAccount = {
+                    'id': f"projects/{project_id}/serviceAccounts/{member[len('serviceAccount:'):]}"
```

Comment:
do we also need to populate other fields as well like name, email, etc? I see we do that for user, group, domain, etc.

Reply:
To load Service Accounts we already used the iam api so in `attach_role_to_service_account` function we need to only match this Service Account and attached to the role